shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,153 @@
+              help=f"Create a square mask from the input file. Return an output nifti file with square mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Input path of the nifti file to mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-size", required=True, type=int, help="Length of the side of the square along first and second dimension"
+                                                     " (in pixels)")
+@click.option("-center", nargs=2, type=int, default=(None, None), help="Center of the cube along first and second "
+                                                                       "dimension (in pixels). If no center is "
+                                                                       "provided, the middle is used.")
+def square(fname_input, output, size, center):
+    """
+            Create a square mask from the input file. Return an output nifti file with square mask.
+
+            Args:
+                fname_input (str): Complete input path of the nifti file to mask.
+                output (str): Output folder for square mask.
+                size (int): Length of the side of the square along first and second dimension (in pixels).
+                center (int): Center of the cube along first and second dimension (in pixels). If no center is
+                            provided, the middle is used.
+                
+            Return:
+                output (str): Filename for the output mask.
+            """
+    # Create the folder where the nifti file will be stored
+    if not os.path.exists(output):
+        os.makedirs(output)
+
+    nii = nib.load(fname_input)
+    data = nii.get_fdata()  # convert nifti file to numpy array
+    fname_mask = os.path.join(output, 'mask.nii.gz')
+
+    if len(data.shape) == 2:
+        mask_sqr = shape_square(data, size, size, center[0], center[1])  # creation of the square mask
+        mask_sqr = mask_sqr.astype(int)
+        nii_img = nib.Nifti1Image(mask_sqr, nii.affine)
+        nib.save(nii_img, fname_mask)
+        click.echo(f"The filename for the output mask is: {os.path.abspath(fname_mask)}")
+        return output
+
+    elif len(data.shape) == 3:
+        for z in range(data.shape[2]):
+            mask_sqr = np.zeros_like(data)  # initialization of 3D array of zeros
+            img_2d = data[:, :, z]  # extraction of a MRI slice (2D)
+            mask_slice = shape_square(img_2d, size, size, center[0], center[1])  # creation of the mask
+            # on each slice (2D)
+            mask_sqr[:, :, z] = mask_slice  # addition of each masked slice to form a 3D array
+            mask_sqr = mask_sqr.astype(int)
+            nii_img = nib.Nifti1Image(mask_sqr, nii.affine)
+            nib.save(nii_img, fname_mask)
+            click.echo(f"The filename for the output mask is: {os.path.abspath(fname_mask)}")
+            return output
```

Comment:
this behaviour should be described in the usage

Reply:
Done now !